native/cocos/scene/RenderScene.cpp

```diff
@@ -79,6 +79,7 @@ void RenderScene::update(uint32_t stamp) {
         if (model->isEnabled()) {
             model->updateTransform(stamp);
             model->updateUBOs(stamp);
+            model->updateOctree();
```

Comment:
octree will be updated many times

Reply:
If world bound is updated,  the model in octree need update, not update the whole octree